Issue #161 : Corrected S3 websites endpoint suffixes and added support for more regions

yashshanker opened the pull request:
Issue #161

Description: 

1. There was a mistake in website endpoint suffixes for S3 Buckets for the following regions:
    - Asia Pacific (Seoul)
    - Asia Pacific (Mumbai)
    - US East (Ohio)
2. Support for the following regions has been added in the template:
    - Canada (Central) 
    - EU (Stockholm)
    - EU (Paris)
    - EU (London)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

yashshanker commented:
@stilvoid Can you kindly review and merge this PR ?

rjlohan commented:
Are the endpoint URLs inconsistent across different regions? I haven't dug into this, but the inconsistency seems off...

yashshanker commented:
The inconsistency in URL endpoints was in the following regions Asia Pacific (Seoul), Asia Pacific (Mumbai), US East (Ohio). I have corrected that in this PR. 
And added support for additional regions, which were not present earlier.
